ec2/spark_ec2.py

```diff
@@ -706,9 +697,7 @@ def wait_for_cluster_state(conn, opts, cluster_instances, cluster_state):
     sys.stdout.flush()
 
     start_time = datetime.now()
-
     num_attempts = 0
-    conn = ec2.connect_to_region(opts.region)
```

Comment:
It opens a connection to EC2. But since we are [already passed in a connection as `conn`](https://github.com/apache/spark/pull/3939#discussion-diff-22621704L693), there is no need to recreate this.